accounting/service.proto

```diff
@@ -50,4 +50,7 @@ message Decimal {
 message BalanceResponse {
     // Carries the amount of funds on the account.
     Decimal balance = 1;
+
+    // Carries response meta information attached by server.
+    service.ResponseMetaHeader ResponseHeader = 99;
```

Comment:
Since there is no `RequestHeader` field in requests, I think `service.ResponseMetaHeader meta_header = 99` is more suitable name for those fields.